src/main/java/net/earthcomputer/clientcommands/command/ReplyCommand.java

```diff
@@ -39,12 +60,13 @@ public static void register(CommandDispatcher<FabricClientCommandSource> dispatc
     }
 
     public static int reply(FabricClientCommandSource source, Component message) throws CommandSyntaxException {
-        if (currentTarget == null) {
+        @Nullable String target = ReplyCommand.getCurrentTarget();
```

Comment:
Nit: usually no need to annotate local variables with `@Nullable`, especially when the IDE is smart enough to infer it as in this case.